### DISCUSSION TITLE
Update company Certificate

### DISCUSSION THREAD

**Discussion by jsaezre** [globaleaks/globaleaks-whistleblowing-software#4174] [Q&A]
Hello,
The certificate (company certificate) of the company is near to expyre. We have the new one but we don´t know the way to change it.,

Our Platform is GlobaLeaks 5.0.1

Could anyone help us? We need the steps we should do for change it.

Thank you

**Comment by evilaliv3** [comment]
Are you using letsencrypt certificates or not?

If you are using letsencrypt globaleaks will take care of renewing your certificate automatically.
If you are using a different certificates to replace the certficate disable the configuration, load your certificate and enable it again.

**Reply by jsaezre** [reply]
Hello,
The certificate is not letsencrypt. Our problem was that we didn't know how update. Now we now that we can use he link https://ourdomain/#/admin/network to upload the certificate in pem format with the private key. The problem is that we had the certficate in p12 format and we need to transform to pem. Anyway now we have it working but really we didn't found any documentation about the way to do it.
Regards for your answer.

Jorge

**Reply by evilaliv3** [reply]
Thank you for this update. We are you managed to solve the situation.

**Comment by evilaliv3** [comment]
tHANK
